Sources/TuistLoader/Models+ManifestMappers/Headers+ManifestMapper.swift

```diff
@@ -1,3 +1,4 @@
+import CryptoKit
```

Comment:
what is this needed for? 🤔

Reply:
hm.. Maybe Xcode automatically added it. Sorry guys, spent a lot of time with investigating why tuist was n't build with some Ruby issues. Will remove it